### ISSUE TITLE
Has anyone trained adaface on WebFace260M?

### ISSUE THREAD

**Issue by martinenkoEduard** [mk-minchul/AdaFace#88]
Has anyone trained adaface on WebFace260M?

**Comment by quanqigu** [comment]
Hi @martinenkoEduard Have you done this?

**Comment by martinenkoEduard** [comment]
> Hi @martinenkoEduard Have you done this?

I tried, but there wasn't enough VRAM on my RTX 3080.
The more classes you have in the output layer, the more memory you need.

**Comment by quanqigu** [comment]
> > Hi @martinenkoEduard Have you done this?
> 
> I tried, but there wasn't enough VRAM on my RTX 3080. The more classes you have in the output layer, the more memory you need.

You are right, don't know if partial fc would help on this. Besides, do you know where can I get webface 260m?

**Comment by martinenkoEduard** [comment]
> > > Hi @martinenkoEduard Have you done this?
> > 
> > 
> > I tried, but there wasn't enough VRAM on my RTX 3080. The more classes you have in the output layer, the more memory you need.
> 
> You are right, don't know if partial fc would help on this. Besides, do you know where can I get webface 260m?

How to enable partial fc? You can enable write a email to - https://www.face-benchmark.org/
They should give you the dataset.

**Comment by mk-minchul** [comment]
PartialFC + AdaFace is implemented in our new face recognition repository https://github.com/mk-minchul/CVLface

**Comment by airkid** [comment]
> PartialFC + AdaFace is implemented in our new face recognition repository https://github.com/mk-minchul/CVLface

Thanks for sharing. I am just working on merging  AdaFace with ParticalFC,  it's very helpful.

**Comment by David594106** [comment]
Hi, what is the process of training AdaFace on WebFace 260M, kindly share the command, thanks

**Comment by wuxuedaifu** [comment]
> > PartialFC + AdaFace is implemented in our new face recognition repository https://github.com/mk-minchul/CVLface
> 
> Thanks for sharing. I am just working on merging AdaFace with ParticalFC, it's very helpful.
Hi, which model did you train for Webface42M particalFC Adaface,   I tried on iresent101 and swinTransformer, the loss stable at 4.5 for iresnet101.  and 38.5 for SwinTransformer KPRPE.  so I doube some problem in the code of SwinTransformereKPRPE

**Comment by wuxuedaifu** [comment]
> PartialFC + AdaFace is implemented in our new face recognition repository https://github.com/mk-minchul/CVLface

Hi, for CVLFace, is float16 training working, I tried several times for Webface42m, but pops out INF & NAN problems several times

**Comment by wensi2690** [comment]
> > PartialFC + AdaFace is implemented in our new face recognition repository https://github.com/mk-minchul/CVLface
> 
> Hi, for CVLFace, is float16 training working, I tried several times for Webface42m, but pops out INF & NAN problems several times

Hi，could you plz share the Webface42m dataset with me? This will be crucial for me, I would gateful for your response!
